### PR TITLE
Feature/logger

### PR DESCRIPTION
**Pull request by rhyslbw**
This is the foundation of the translation from [Job Collection's eventEmitter](https://github.com/vsivsi/meteor-job-collection#jcevents---server) structure to Space.messaging.Event/s, and the separate eventSubscribing logger. It looks a little verbose but it will be advantageous for each event type to be fully independent as I flesh out the critical information with the upcoming tasks.

Job Collection has good test coverage, so I'm only testing the operations that I can easily considering we're working fast and most is just a simple mapping.

### CODE REVIEW

**Comment by DominikGuzei** [comment]
I would stick to the dash-filenaming-convention :wink:

File: source/server/event_publisher.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  dependencies: {
+    jobCollection: 'Space.jobQueue.Jobs'
+  },
+
+  onDependenciesReady() {
+    var self = this;
+    this.jobCollection.events.on('call', function (message) {
+      switch(message.method) {
+        case 'startJobServer':
+          self._onStartJobServer(message);
```
Comment (DominikGuzei) [inline source/server/event_publisher.js]: i would just publish the events directly in the switch statement – the methods don't add much value here

File: source/server/logger.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  },
+
+  eventSubscriptions() {
+    return [
+      {
+        'Space.jobQueue.JobServerStarted'() {
+          this.log.info("Job Server Started");
+        }
+      },
+      {
```
Comment (DominikGuzei) [inline source/server/logger.js]: you don't need an object for each event type! you can just return an array with a single object and the event names as keys like this:

``` javascript
return [{
  'Space.jobQueue.JobServerStarted'() {
    this.log.info("Job Server Started");
  },
  'Space.jobQueue.JobServerShutdown'() {
    this.log.info("Job Server shutdown");
  },
  // …
}]
```

**Comment by DominikGuzei** [comment]
Looks great apart from the minor stylistic issues :wink: looking forward to see this in action!

File: source/server/event_publisher.js (hunk 1 above)
Comment (rhyslbw) [inline source/server/event_publisher.js]: :+1:

File: source/server/logger.js (hunk 2 above)
Comment (rhyslbw) [inline source/server/logger.js]: :+1:

**Pull request merged**